js/packages/cli/src/helpers/upload/ipfs.ts

```diff
@@ -43,7 +43,7 @@ export async function ipfsUpload(
   const manifestJson = JSON.parse(manifestBuffer.toString('utf8'));
   manifestJson.image = mediaUrl;
   manifestJson.properties.files = manifestJson.properties.files.map(f => {
-    return { ...f, uri: mediaUrl };
+    return { ...f, src: mediaUrl };
```

Comment:
@21e8 was this change necessary?

The Metaplex Token Metadata standard says it should be `uri`.
I've used `src` in arweave-bundles upload as well, changing it back to `uri`.

Reply:
FYI, dropped this change.